docs/architecture/adr-004-tips-contract.md

```diff
@@ -54,26 +55,31 @@ tip and paid to the contract.
 ```rust
 pub enum ServiceFee {
   Fixed {amount: Vec<Coin>},
-  Percentage{value: u64}
+  Percentage{value: u64, decimals: u32}
 }
 ```
 
-* The value of the percentage is represented by a `u64` cause float can't be used inside contracts. The Service fee will then be calculated as follow so:
-  * Service Fee = `tip_amount * value/100`
+The `Percentage` fee is represented by a:
+  * `value` field that identifies the number of percentage the admin applies (e.g. value = 2 means that the percentage can be 2%, 20%, etc..based on the 2nd field);
```

Comment:
```suggestion
  * `value` field that identifies the number of percentage the admin applies (e.g. value = 2 means that the percentage can be 2%, 0,2%, etc..based on the 2nd field);
```
I think that having 0,2% as second example makes more clear the meaning of `decimals`